.zenodo.json

```diff
@@ -0,0 +1,51 @@
+{
+  "creators": [
+    {
+      "name": "Huard, David",
+      "affiliation": "Ouranos",
+      "orcid": "0000-0003-0311-5498"
+    },
+    {
+      "name": "Smith, Trevor James",
+      "affiliation": "Ouranos",
+      "orcid": "0000-0001-5393-8359"
+    },
+    {
+      "name": "Arsenault, Richard",
+      "affiliation": "École de technologie supérieure",
+      "orcid": "0000-0003-2834-2750"
+    },
+    {
+      "name": "Mai, Juliane",
+      "affiliation": "University of Waterloo",
+      "orcid": "0000-0002-1132-2342"
+    },
+    {
+      "name": "Vu, Tien Long",
```

Comment:
That's not my name, it's Tuan, not Tien :D  I'll open a PR ...

Reply:
Shoot. Sorry!